controllers/controllers.js

```diff
@@ -27,7 +27,28 @@ exports.getArticleByID = (req, res, next) =>{
 
 exports.getAllArticles = (req, res, next) =>{
     return fetchAllArticles().then((queryResponse)=>{
-        console.log(queryResponse)
         res.status(200).send({articles: queryResponse})
     })
+}
+
+exports.getArticleComments = (req, res, next) =>{
+   const doesArticleExist= checkArticleExists(req.params.article_id)
+
+   const retrieveComments = fetchArticleComments(req.params.article_id)
+  
+
+   return Promise.all([doesArticleExist, retrieveComments]).then((queryResponse)=>{
+        const [,comments] = queryResponse
+        
+        if(comments.length === 0){
+            res.status(200).send({msg: 'No Comments'})
```

Comment:
If there are no comments, you are best off just sending back an empty array, not only are you changing the data type which would be more effort on the frontend to change, but on the front end, you can make a decision what to display to the users based on the length of the array.